Easydict/NewApp/Utility/Extensions/QueryService+ConfigurableService/OpenAIService+ConfigurableService.swift

```diff
@@ -21,12 +21,6 @@ extension EZOpenAIService: ConfigurableService {
                 key: .openAIAPIKey,
                 placeholder: "service.configuration.openai.api_key.placeholder"
             )
-            // endpoint
-            ServiceConfigurationInputCell(
-                textFieldTitleKey: "service.configuration.openai.endpoint.title",
-                key: .openAIEndPoint,
-                placeholder: "service.configuration.openai.endpoint.placeholder"
-            )
```

Comment:
Although a new one-api service has been added this time, I don't think we should remove the endpoint from the previous OpenAI, which will cause the existing configuration service to not work.

<img width="453" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/783243f2-9d65-4552-96ab-b6f5bcf133e0">

Reply:
But keeping this will make there minor differences between the new service and the OpenAI service.

Reply:
ok, already reverted the changes of endPoint part.